clang-tools-extra/clangd/InlayHints.cpp

```diff
@@ -709,7 +738,8 @@ class InlayHintVisitor : public RecursiveASTVisitor<InlayHintVisitor> {
 private:
   using NameVec = SmallVector<StringRef, 8>;
 
-  void processCall(Callee Callee, llvm::ArrayRef<const Expr *> Args) {
+  void processCall(Callee Callee, SourceRange RParenOrBraceRange,
```

Comment:
I think it would be clearer for the `SourceRange RParenOrBraceRange` parameter to be `SourceLocation RParenOrBraceLoc` instead.

Both call sites in fact pass a `SourceLocation`, which currently gets implicitly converted to a singleton range.

I'd rather we do that conversion inside the function's implementation, and make it explicit with `SourceRange{RParenOrBraceLoc}`.